flux-fixpoint/src/lib.rs

```diff
@@ -198,9 +200,9 @@ impl fmt::Display for KVar {
     }
 }
 
-impl fmt::Display for UifDef {
+impl fmt::Display for ConstInfo {
     fn fmt(&self, f: &mut fmt::Formatter<'_>) -> fmt::Result {
-        write!(f, "(constant {} {})", self.name, self.sort)
+        write!(f, "(constant {0:?} {1:?})", self.name, self.sort)
```

Comment:
```suggestion
        write!(f, "(constant {:?} {:?})", self.name, self.sort)
```

Reply:
Actually, I just thought we can add a comment here to know what's the origin the same way we do for kvars.

```suggestion
        write!(f, "(constant {:?} {:?}) // orig: {}", self.name, self.sort, self.orig)
```